keras_hub/src/layers/preprocessing/image_converter.py

```diff
@@ -97,7 +97,8 @@ def __init__(
         image_size=None,
         scale=None,
         offset=None,
-        crop_to_aspect_ratio=True,
```

Comment:
We should not change the default on this. One it would be breaking, and two, resizing changing aspect ratio is not a great default.

Reply:
Done